README.md

```diff
@@ -6,6 +6,7 @@ We use a custom homebrew 'tap' for a couple of different cases:
 
 * qt and qt-webkit - we build our own custom versions of qt and qtwebkit to support some proprietary features for capturama
 * postgres - homebrew aggressively upgrades postgres, and we want to keep everyone nearer to the same version
+* nsq - we maintain a very old version of NSQ that is incompatible with NSQ that is available in homebrew today
```

Comment:
The only note I may make is that this version is compatible with prod, though still newer. I'm not sure if that matters for this README or the individual projects.

Reply:
I think this is better here, given the way homebrew "versions" the only place the version and compatibility info matter is in this repo where we define what version we're maintaining.